tsconfig.json

```diff
@@ -26,6 +26,11 @@
 
     "noErrorTruncation": true
   },
-  "files": [],
-  "include": []
+  "include": [
```

Comment:
`apps` should not be in the root build, there are `tsconfig`s specifically for them

Reply:
yeah thats fine, i dont think i actually wanted to check that in, its not even required for the serverless build